Code/Tools/AssetProcessor/AssetBuilderSDK/AssetBuilderSDK/AssetBuilderSDK.cpp

```diff
@@ -1612,4 +1614,72 @@ namespace AssetBuilderSDK
     {
         return m_errorsOccurred;
     }
+
+    AZ::u64 GetHashFromIOStream(AZ::IO::GenericStream& readStream, AZ::IO::SizeType* bytesReadOut, int hashMsDelay)
+    {        
+        constexpr AZ::u64 HashBufferSize = 1024 * 64;
+        char buffer[HashBufferSize];
+
+        constexpr bool ErrorOnReadFailure = true;
```

Comment:
Unused variable

Reply:
Yes. It was captured in AR and it's fixed now.